README.md

```diff
@@ -1,7 +1,19 @@
 # Adyen Module
 
+[![Build Status](https://travis-ci.org/spryker-eco/adyen.svg?branch=master)](https://travis-ci.org/spryker-eco/adyen)
+[![License](https://img.shields.io/github/license/spryker-eco/adyen.svg?b=master)](https://github.com/spryker-eco/adyen)
+
 ## Installation
 
 ```
 composer require spryker-eco/adyen
 ```
+
+## Documentation
+
+[TBD](https://academy.spryker.com)
+
+## Code checks
+
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/spryker-eco/adyen/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/spryker-eco/adyen/?branch=master)
+[![Build Status](https://scrutinizer-ci.com/g/spryker-eco/adyen/badges/build.png?b=master)](https://scrutinizer-ci.com/g/spryker-eco/adyen/build-status/master)
```

Comment:
They are just added to the top badges as well

Reply:
Moved.